package.json

```diff
@@ -59,6 +60,7 @@
     "redux": "^4.1.1",
     "request-promise": "^4.2.6",
     "sleep-promise": "^9.1.0",
+    "taffydb": "^2.7.3",
```

Comment:
Are we using temp DB's? Is this intended?

Reply:
@Jeevan-Kishore not intentional, we will delete. Thanks for spotting it